docker/init-db.sql

```diff
@@ -0,0 +1,41 @@
+-- Create organizations table
+CREATE TABLE organizations (
+                               id SERIAL PRIMARY KEY,
+                               organization_id VARCHAR(255) NOT NULL,
+                               name VARCHAR(255),
+                               address VARCHAR(255)
+);
+
+-- Insert dummy organizations
+INSERT INTO organizations (organization_id, name, address) VALUES
+                                                               ('ai4hf_aumc', 'Amsterdam UMC', 'Address of Amsterdam UMC');
+
+-- Create personnel table
+CREATE TABLE personnel (
+                           id SERIAL PRIMARY KEY,
+                           person_id VARCHAR(255) NOT NULL,
+                           organization_id VARCHAR(255) REFERENCES organizations(organization_id),
```

Comment:
This script gives the following error: ERROR:  there is no unique constraint matching given keys for referenced table "organizations"

If we get rid of the organizationId field in organizations table and utilize the id field instead, then this error can disappear. Please make sure that you can start the docker services from the scratch. (i.e. delete all containers and volumes and try to start services, they should start without an error)